hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -148,16 +148,18 @@
     {
       name: build_seed
       pre_build_cmds: [
-        // TODO: find where in run phase we are using this pkg. It fails during ibex TLUL integrity
-        // check.
-        // '''cd {proj_root} && ./util/topgen.py -t {ral_spec} \
-        //        -o hw/top_earlgrey --rnd_cnst_seed {seed}
-        // ''',
+        // Generate RndCnsts
+        '''cd {proj_root} && ./util/topgen.py -t {ral_spec} \
+               -o hw/top_earlgrey --rnd_cnst_seed {seed}
+        ''',
+        // Generate LC encoding
+        "cd {proj_root} && ./util/design/gen-lc-state-enc.py --seed {seed}",
+        // Generate OTP memory map and scrambling constants keys.
         "cd {proj_root} && ./util/design/gen-otp-mmap.py --seed {seed}",
         // Use eval_cmd to save build_seed in a file and reuse that file during run phase.
```

Comment:
I think this comment can be removed now?

Reply:
Good point - I updated the comment.